pkgs/development/python-modules/oca-port/default.nix

```
@@ -0,0 +1,45 @@
+{
+  buildPythonPackage,
+  click,
+  fetchPypi,
+  GitPython,
+  lib,
+  nix-update-script,
+  requests,
+  setuptools-scm,
+  setuptools,
+}: let
+  moduleName = "oca_port";
+in
+  buildPythonPackage rec {
+    pname = "oca-port";
+    version = "0.12";
+
+    src = fetchPypi {
+      inherit version;
+      pname = moduleName;
+      sha256 = "sha256-adwqnhe5BLo5tsjaLb0XHlTFxSKNutsb6hlTXjA4Wio=";
```

```suggestion
      hash = "sha256-adwqnhe5BLo5tsjaLb0XHlTFxSKNutsb6hlTXjA4Wio=";
```
The SRI hash is preferred.